Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAgentNPMConfig.py

```diff
@@ -126,7 +150,11 @@ def Set_Marshall(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
     recvdContentChecksum = md5.md5(Contents).hexdigest().upper()
     if recvdContentChecksum != ContentChecksum:
         # data is corrupt do not proceed further
-        LOG_ACTION.log(LogType.Error, 'Content received did not match checksum, exiting Set')
+        LG().Log(LogType.Error, 'Content received did not match checksum, exiting Set')
+        return [-1]
+    if not os.path.exists(os.path.join(PREFIX_ETC, ConfigID)):
```

Comment:
If ConfigID is None here then this will raise an AttributeError. Better to verify this condition in regular SET call below, after the init_vars kicks in.